code/Controllers/OldPageRedirector.php

```diff
@@ -29,7 +29,10 @@ public function onBeforeHTTPError404($request)
         $getvars = $request->getVars();
         unset($getvars['url']);
 
-        $page = static::find_old_page($params);
+        $page = self::find_old_page($params, 0);
```

Comment:
Just FYI: It looks like this use of `self::` here instead of `static::` introduced a regression.  that was supposed to be addressed in #2478 (which itself was fixing #2438).